Organic Impressions Estimator/RankingDataFetcher.js

```diff
@@ -188,15 +210,22 @@ function updateRankingSheet(newRankingData, competitorAsins) {
  */
 function updateRankByDaySheet(newRankingData) {
   const ss = SpreadsheetApp.openById(TARGET_SPREADSHEET_ID);
-  const rankByDaySheet = ss.getSheetByName('Rank by Day');
+  let rankByDaySheet = ss.getSheetByName('Rank by Day');
+
+  // Create the 'Rank by Day' sheet if it doesn't exist
```

Comment:
Just an edge case I discovered in QA -- if you delete the "Rank by Day" sheet, then click "Get Ranks" it'll create the sheet but won't populate the Rank by Day sheet with the data that's already sitting in the Raw Rank Data sheet.